### PR TITLE
Support elevenlabs 2.5

### PR DESCRIPTION
**Pull request by xquanluu**

### CODE REVIEW

File: lib/models/speech-credential.js [hunk 1]
```diff
@@ -23,10 +23,11 @@ class SpeechCredential extends Model {
   static async getSpeechCredentialsByVendorAndLabel(service_provider_sid, account_sid, vendor, label) {
     let sql;
     if (account_sid) {
-      sql = `SELECT * FROM speech_credentials WHERE account_sid = ? AND vendor = ? ${label ? 'AND label = ?' : ''}`;
+      sql = `SELECT * FROM speech_credentials WHERE account_sid = ? AND vendor = ?
+        AND label ${label ? '= ?' : 'is NULL'}`;
     } else {
       sql = `SELECT * FROM speech_credentials WHERE service_provider_sid = ? AND vendor = ?
-        ${label ? 'AND label = ?' : ''}`;
+        AND label ${label ? '= ?' : 'is NULL'}`;
```
Comment (davehorton) [inline lib/models/speech-credential.js:30]: is this a fix for a different bug?

Comment (xquanluu) [reply lib/models/speech-credential.js:30]: Yes this is for fixing issue that, When we ask tts and stt model for vendor elevenlabs without any label

Api server will run a query without checking label, so the query might return a list of available speech credential for same vendor, api server then choose the 1st object in the list which is wrong to do next task

When no label is provided, we just want to take speech credential with vendor is NULL

Comment (davehorton) [reply lib/models/speech-credential.js:30]: there was a similar issue reported but using whisper tts.  Would this fix that as well?

Comment (xquanluu) [reply lib/models/speech-credential.js:30]: I think yes, could you link the issue to this PR?

Comment (davehorton) [reply lib/models/speech-credential.js:30]: https://jambonz.freshdesk.com/a/tickets/296

Comment (xquanluu) [reply lib/models/speech-credential.js:30]: No this issue is maybe something on feature server and not this play.

**Review by davehorton — APPROVED** [review]

**Pull request merged**